core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/worker/neo/WorkerRPCHandlerInitializer.scala

```diff
@@ -6,18 +6,19 @@ import edu.uci.ics.amber.engine.architecture.worker.neo.promisehandlers.PauseHan
 import edu.uci.ics.amber.engine.common.ambermessage.WorkerMessage.ExecutionPaused
 import edu.uci.ics.amber.engine.common.ambertag.neo.VirtualIdentity.ActorVirtualIdentity
 import edu.uci.ics.amber.engine.common.promise.{
-  PromiseHandlerInitializer,
-  ControlInvocation,
-  PromiseManager,
+  RPCClient,
+  RPCHandlerInitializer,
+  RPCServer,
   WorkflowPromise
 }
 import edu.uci.ics.amber.engine.common.statetransition.WorkerStateManager
 
-class WorkerPromiseHandlerInitializer(
+class WorkerRPCHandlerInitializer(
```

Comment:
For this and RPCHandlerInitializer class, please add a comment to say how to add a handler. What is the expected input and output

Reply:
done